docs/reference/tctl/namespace/register.md

```diff
@@ -0,0 +1,151 @@
+---
+id: register
+title: tctl namespace register
+sidebar_label: register
+description: How to register a Namespace using tctl.
+tags:
+  - reference
+  - tctl
+---
+
+The `tctl namespace register` command registers a [Namespace](/docs/content/what-is-a-namespace).
+
+`tctl namespace describe`
+
+The following modifiers control the behavior of the command.
+
+### `--active-cluster`
+
+How to specify the name of the active [Temporal Cluster](https://docs.temporal.io/docs/content/what-is-a-temporal-cluster/) when registering a [Namespace](/docs/content/what-is-a-namespace) in tctl.
+
+Alias: `--ac`
+
+**Example**
+
+```
+tctl namespace register --active-cluster <name>
+```
+
+### `--clusters`
+
+How to specify a list of [Temporal Clusters](https://docs.temporal.io/docs/content/what-is-a-temporal-cluster/) when registering a [Namespace](/docs/content/what-is-a-namespace) in tctl.
+
+The list contains the names of Clusters (separated by spaces) to which the Namespace can fail over.
+
+This modifier is valid only when the `--global-namespace` modifier is set to true.
+
+Alias `--cl`
+
+**Example**
+
+```
+tctl namespace register --clusters <names>
+```
+
+### `--description`
+
+How to specify a description when registering a [Namespace](/docs/content/what-is-a-namespace) in tctl.
+
+Alias `--desc`
+
+**Example**
+
+```
+tctl namespace register --description <value>
+```
+
+### `--global-namespace`
+
+How to specify whether a [Namespace](/docs/content/what-is-a-namespace) is a Global Namespace.
+
+Alias `--gd`
```

Comment:
Wouldn't the Alias derive to `--gn` for _g_lobal _n_amespace?

Reply:
Makes sense, but the code sez otherwise: https://github.com/temporalio/tctl/blob/316cdd28256dab4c850c56542efb70e426e78031/cli/flags.go#L149